ivadomed/config/config_vertebral_labeling.json

```diff
@@ -0,0 +1,103 @@
+{
+    "command": "train",
+    "gpu": 7,
+    "log_directory": "labeling_t1",
+    "debugging": true,
+    "loader_parameters": {
+        "bids_path": "/home/GRAMES.POLYMTL.CA/luroub/luroub_local/lurou_local/deep_VL_2019/straight/",
+        "target_suffix": ["_heatmap"],
+        "roi_params": {
+            "suffix": null,
+            "slice_filter_roi": null
+        },
+        "contrast_params": {
+            "training_validation": ["T1w_straight_mid"],
+            "testing": [ "T1w_straight_mid"],
+            "balance": {}
+        },
+        "slice_filter_params": {
+            "filter_empty_mask": false,
+            "filter_empty_input": true
+        },
+        "slice_axis": "sagittal",
+        "multichannel": false,
+	"soft_input": true
```

Comment:
Before merging, may I ask you to add this param in the other config files? many thanks

Reply:
Done